### ISSUE TITLE
Colmap problem for nerfbusters_dataset 

### ISSUE THREAD

**Issue by Pamimi** [ethanweber/nerfbusters#12]
<img width="1053" alt="image" src="https://github.com/ethanweber/nerfbusters/assets/89392940/50bd03fc-17c4-4fe2-ae01-196bc6dc747d">
When I used the code of nerfstudio to perform colmap reconstruction on the videos of the pikachu training set in nerfbuster-capture, I encountered a reconstruction failure.
(The reason for this is because I found that the point3D sparse point cloud of the preprocessed data is constructed simultaneously through the training set and validation set)
Could you please tell me how you built the data set to ensure that all images are registered?

**Comment by ethanweber** [comment]
Hi @Pamimi, can you have a look at this? [https://docs.nerf.studio/quickstart/custom_dataset.html#training-and-evaluation-on-separate-data](https://docs.nerf.studio/quickstart/custom_dataset.html#training-and-evaluation-on-separate-data) You'll need to pass in `--eval-data`. We pass in the training and evaluation videos at the same time, which should make COLMAP succeed. Hope this helps!
